rs/registry/canister/src/mutations/do_remove_node_operators.rs

```diff
@@ -36,6 +34,18 @@ impl Registry {
                     })
             })
             .collect();
+        let valid_node_operator_ids = payload
+            .node_operators_to_remove
+            .into_iter()
+            .filter(|node_operator_id| {
+                let node_operator_record_key =
+                    make_node_operator_record_key(*node_operator_id).into_bytes();
+                self.get(&node_operator_record_key, self.latest_version())
+                    .is_some()
+            })
+            .collect();
+        let mut valid_node_operator_ids =
+            [valid_node_operator_ids_from_blobs, valid_node_operator_ids].concat();
```

Comment:
We could probably make this exclusive.  Nobody is going to be sending these in both ways.  If the new one is present, we could panic if they send values in the old way.

Reply:
Is there some benefit to making it exclusive?

Reply:
Avoiding confusing semantics.  Two API fields shouldn't mean the same thing, but if they do, then there should be some rhyme or reason to when you should use one instead of the other.

In this case, we're making an improvement to the API but have to support the old way of doing it. 

IMO, if we want to encourage clients to use the new way, we should not let them use both.  Eventually we can remove the old way.

Reply:
No Change Required here if it's a lot of trouble, I just think it would be better not to mix and match them.

Reply:
Understood, I adjusted the code.